src/System.Console/src/System/IO/SyncTextReader.cs

```diff
@@ -11,7 +11,6 @@ namespace System.IO
     internal sealed class SyncTextReader : TextReader
     {
         private readonly TextReader _in;
-        private readonly object _methodLock = new object();
```

Comment:
:bulb:  I'd rather assign `_methodLock = this;` with comment explaining why we lock on this.